uspto/process_uspto.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Download HF Dataset repo
+git clone git@hf.co:datasets/baber/USPTO ./data/uspto/raw
+
+# Get the current directory
+CURRENT_DIR=$(basename "$PWD")
+
+# Check if we are already in the uspto directory
+if [ "$CURRENT_DIR" != "uspto" ]; then
+    # Navigate to the uspto directory
+    cd uspto || exit
+fi
+
+
+# Clone MathML to LaTeX converter
+git clone https://github.com/baberabb/mathml-to-latex.git
+
+# Navigate to the Node.js project directory
+cd mathml-to-latex || exit
+
+# Install Node.js dependencies
+npm install
```

Comment:
I think we should segment this into two scripts, one that sets up the mathml-to-latex server (does the clone, install, etc) and one that runs data processing (start the server, run the code, kill the server, etc)

I'll do a more through review later today